gateway/utils/metrics/projects.go

```diff
@@ -97,7 +98,7 @@ func (m *Module) generateMetricsRequest(project *config.Project, ssl *config.SSL
 func (m *Module) updateSCMetrics(clusterID, projectID string, set, min map[string]interface{}) {
 	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
 	defer cancel()
-	result, err := m.sink.Upsert("config_metrics").Where(types.And(types.Cond("id", "==", clusterID), types.Cond("project", "==", projectID))).Set(set).Min(min).Apply(ctx)
+	result, err := m.sink.Upsert("config_metrics").Where(types.And(types.Cond("cluster_id", "==", clusterID), types.Cond("project", "==", projectID))).Set(set).Min(min).Apply(ctx)
```

Comment:
Why don't you just put the where clause on the id field?

Reply:
ok